test/test_operations.py

```diff
@@ -1826,6 +1826,16 @@ def test_view_and_copy_(self):
     y[::2].copy_(x[::2])
     self.assertEqual(y, [1, 0, 3, 0, 5, 0])
 
+  def test_view_and_multi_mark_step(self):
+    xla_device = xm.xla_device()
+    t1 = torch.zeros(100, device=xla_device)
+    t1[10] = 113
+    xm.mark_step()
+    t1[12] = 1123
+    xm.mark_step()
+    self.assertNotIn('update_slice',
```

Comment:
Wouldn't the graph be trimmed anyway after `mark_step()`? I just tried on my branch without this patch, and it doesn't have the `update_slice` after. But maybe I am not following / understanding 🥟 

nit. It might be helpful to add a brief comment explaining what this assertion is about -- esp for the future readers who haven't been following the original issue.

Reply:
hmm, without this change, the test failed at my end.

Reply:
The bug is that the second `mark_step` won't execute the IR of `t1`, can you verify?

Reply:
Ugh, it failed -- sorry for the false alarm. I can verify that the graph wasn't trimmed after the second `mark_step`.